package.json

```diff
@@ -205,6 +205,7 @@
     "minimist": "1.2.0",
     "mocha-sinon": "1.2.0",
     "npm-run-all": "4.1.5",
+    "nyc": "13.1.0",
     "rimraf": "2.6.2",
     "sinon": "4.5.0",
     "standard-version": "4.4.0",
```

Comment:
@PascalPrecht `test.js` is being loaded here, as part of the tests for `ProcessWrapper`:

https://github.com/embark-framework/embark/blob/master/src/test/processLauncher.js#L23

The test works correctly whether or not the file exists, but if the file doesn't exist then we get error-output in the test. I think not having the error output is preferable because when seeing it zip by I reflexively think maybe a test failed.

We could include a comment-line in the otherwise empty `test.js` explaining why it's there. We could also rename it to something like `dummy.js`. Thoughts?

Reply:
@michaelsbradleyjr now that you point this out, I actually think, the tests should fail when the given `modulePath` for `ProcessWrapper` doesn't exist.

Reply:
So I guess my vote here would be:

1. Yes, keep `test.js` and ideally name it to something more meaningful as it's really just a test dummy.

2. Make  `ProcessWrapper` throw (and exit?) when `modulePath` doesn't exist.